account_multicurrency_revaluation/wizard/wizard_currency_revaluation.py

```diff
@@ -233,9 +208,9 @@ def create_move():
                          'period_id': period.id,
                          'date': form.revaluation_date,
                          'to_be_reversed': reversable}
-            return move_obj.create(cr, uid, base_move, context=context)
+            return move_obj.create(base_move).id
 
-        def create_move_line(move_id, line_data, sums):
+        def create_move_line(line_data, sums):
```

Comment:
We do not need to pass self here as the function is a nested one, so self is already accessible. And self is not used in the function create_move_line.